src/models.rs

```diff
@@ -229,15 +272,29 @@ impl Default for TimeEntry {
 impl std::fmt::Display for TimeEntry {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         let summary = format!(
-            "[{}]{} –  {} {}",
+            //{$/space} [{duration}]{running indicator/space} - {description/No Description}{@project/empty string} {#tags/empty string}
```

Comment:
For future it might be neat to implement Display for the `Vec<TimeEntry>` and peek into the list to do tabular formatting.